### ISSUE TITLE
Mistake in name of Ukrainian language

### ISSUE THREAD

**Issue by Mant1kor** [SimpleMachines/SMF#7566]
#### Description
While translating SMF to Ukrainian, I've noticed that name of the language has a mistake.
Current name — "Ukranian", should be — "Ukra**i**nian". (proof - [ISO 639-2 Language Code List](https://www.loc.gov/standards/iso639-2/php/code_list.php))

The mistake exists in Crowdin builds, in file names
![зображення](https://user-images.githubusercontent.com/5281753/196748156-45f91b92-41ef-4ae7-87d9-4a761daa10bf.png)
![зображення](https://user-images.githubusercontent.com/5281753/196748186-b63441ce-dbea-4bd4-aef7-ffed0eb56219.png)

and I've found it in one file
https://github.com/SimpleMachines/SMF/blob/71c73f5da56932c1269d1f12356aca986532978f/other/upgrade.php#L2992
### Steps to reproduce
1. Install Ukrainian language pack.
2. View language and file names.

### Environment (complete as necessary)
- Version/Git revision: 2.1.2
- Database Type: MySQL
- Database Version: 8.0.28
- PHP Version:	8.0.20

**Comment by m4z** [comment]
The bug isn't in Crowdin, that correctly uses "[Ukrainian](https://crowdin.com/project/simple-machines-21-beta/uk)", so maybe it's in our script that creates the language packs. However, I just looked at a copy of `smf_section_languages.sql` from 2021-02 and that also correctly uses (used?) `(48, 'ukrainian', 1, 'uk'),`. The [language downloads](https://download.simplemachines.org/index.php?languages;id=48) also have "ukrainian" on the site and in the files.

I guess I'm late to the party and somebody already fixed stuff. (╯°□°)╯︵ ┻━┻

**Comment by Mant1kor** [comment]
> The [language downloads](https://download.simplemachines.org/index.php?languages;id=48) also have "ukrainian" on the site and in the files.

The version 2.0.19 (from 21.12.2021) has correct name of the language, but latest version 2.1.2 still has the mistake.
Seems like the name of the language was correct until some period, and after that the mistake appeared.
So, @m4z, you're welcomed to investigate and fix this issue if you want to.

**Comment by m4z** [comment]
You're correct, the braniac that I am at this hour, I just looked at a 2.0.19 download.

@Dzonny or @jdarwood007 have better visibility into the inner workings of the language pack generation mechanism than I do, so I'll let them have a go at it. :wink:

**Comment by jdarwood007** [comment]
Crowdin is providing the files named that way.  From the export we are getting files like this:
```
➜  ~ ls ~/Downloads/crowdinBuild/uk-UA/Themes/default/languages
Admin.ukranian.php                Errors.ukranian.php               ManageMail.ukranian.php           ManageSettings.ukranian.php       Packages.ukranian.php             Settings.ukranian.php
Agreement.ukranian.php            Help.ukranian.php                 ManageMaintenance.ukranian.php    ManageSmileys.ukranian.php        PersonalMessage.ukranian.php      Stats.ukranian.php
Alerts.ukranian.php               Install.ukranian.php              ManageMembers.ukranian.php        Manual.ukranian.php               Post.ukranian.php                 Themes.ukranian.php
Drafts.ukranian.php               Login.ukranian.php                ManagePaid.ukranian.php           ModerationCenter.ukranian.php     Profile.ukranian.php              Timezones.ukranian.php
Editor.ukranian.php               ManageBoards.ukranian.php         ManagePermissions.ukranian.php    Modifications.ukranian.php        Reports.ukranian.php              Who.ukranian.php
EmailTemplates.ukranian.php       ManageCalendar.ukranian.php       ManageScheduledTasks.ukranian.php Modlog.ukranian.php               Search.ukranian.php               index.ukranian.php
```

I found the location in Crowdin (Custom Language Mapping) that had it misspelled.  It will fix up after it updates on the next cycle.

**Comment by m4z** [comment]
This can be closed, the current 2.1.x language pack download contains the fix. :+1:

**Comment by jdarwood007** [comment]
Only note is that you will have 2 files now.  One with the misspelling and the fixed one.  Delete the old one.
You can update all the members with:
```
UPDATE {db_prefix}members
SET lngfile = 'ukrainian'
WHERE lngfile = 'ukranian'
```
